app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= text_f f, :url, :size => "col-md-8", :help_block => _("e.g. https://ovirt.example.com/api"), :help_inline => documentation_button('5.2.7oVirt/RHEVNotes') %>
 <%= text_f f, :user, :help_block => _("e.g. admin@internal") %>
 <%= password_f f, :password, :unset => unset_password? %>
+<%= checkbox_f f, :filtered_api, :help_inline => _("Use Filter header which makes API limited and slower but is available to non-admin users."), :label => _('User account') %>
```

Comment:
the problem is that it present a follow up problems, for example:
- if you are a large env, and has a lot of hosts, the vm list might not work
- associate vms will not work as well either (both listing the vms).

this also limit us to support better leverage ovirt, for example, we want
for associate vms to use search to find relevant mac addresses instead of
in memory searching for all macs, since search is not exposed, we would not
be able to use that.
I wonder if the better way will be to ask for an admin account - note that
in oVirt, admin does not have a full permissions (e.g. superuser) just
labeled as admin

I also don't like this, but I'm worried that a limitation on oVirt side
will create confusion for our users.